### PR TITLE
refactor(theme): align text styles used in app with `Figma` design specifications

### PR DESCRIPTION
**Pull request by esmaeil-ahmadipour**

### CODE REVIEW

**Comment by esmaeil-ahmadipour** [comment]
Pouria , please check [some errors](https://github.com/pactus-project/pactus-gui/actions/runs/12198536786/job/34030496588?pr=37) on `ci` and fix them. then send review request again.

---
 @PouriaMoradi021

**Comment by PouriaMoradi021** [comment]
everything is done!
----------
@esmaeil-ahmadipour

**Comment by phoenixit99** [comment]
> ## everything is done!
> @esmaeil-ahmadipour

The PR is Draft. Please change it to "Ready for review"

**Comment by esmaeil-ahmadipour** [comment]
@phoenixit99  , @PouriaMoradi021  


Please note that the final commit must be made after the latest changes and right before the merge. This commit should include the changelog updates and the app version upgrade. We need to either have a discussion to align on this or follow the established format used in the changelog file so far. The commit message should also follow a consistent pattern.

```
update: update `CHANGELOG.md` & `pubspec.yaml` files
```

**Review by phoenixit99 — APPROVED** [review]

**Review by phoenixit99 — APPROVED** [review]
Review done

**Comment by esmaeil-ahmadipour** [comment]
please use `onSurfacePallet` in text styles . according some part of material spec in [this link](https://m3.material.io/styles/color/roles#982366a8-9f4e-4eef-94d6-e0e27d07226f).

> Surface – Default color for backgrounds
> 
> On surface – Text and icons against any surface color

Examining Figma, we conclude that there is no consistent color for texts in light and dark mode, and there is a noticeable irregularity. Due to the need to temporarily fix this issue, follow the solution below :

Light Text Color : `#242424`

Dark  Text Color `#FFFFFF`

Please add this color for color palettes (surface and non-surface) - if they are not available -

---

@PouriaMoradi021

**Comment by esmaeil-ahmadipour** [comment]
**Document the Text Style Names**
Ensure you document the name of each text style above it. This will make it easier to locate and reference them.

---

@PouriaMoradi021

File: lib/src/core/constants/inter_text_styles.dart [hunk 1]
```diff
@@ -0,0 +1,410 @@
+import 'package:flutter/material.dart' show FontWeight, TextStyle;
+
```
Comment (PouriaMoradi021) [inline lib/src/core/constants/inter_text_styles.dart:2]: test 
@esmaeil-ahmadipour

Comment (PouriaMoradi021) [inline lib/src/core/constants/inter_text_styles.dart:2]: test 
@esmaeil-ahmadipour

**Pull request merged**